doxia-modules/doxia-module-apt/src/test/java/org/apache/maven/doxia/module/apt/AptParserTest.java

```diff
@@ -486,6 +486,24 @@ public void testLiteralAnchor() throws Exception {
         assertSinkEquals(it, "link_", "sectionTitle1_", "section1_", "body_");
     }
 
+    @Test
+    public void testMultipleAuthors() throws Exception {
+        // DOXIA-691
```

Comment:
This line is redundant since we have "git blame"

Reply:
It is the pattern used in all test methods, so I would say conformity over redundancy